### PR TITLE
fix(scripts/artifacts): fix check-tiup.sh

### PR DESCRIPTION
**Pull request by wuhuizuo**
Compatibility to run for version lower then v8.4.0

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

**Comment by wuhuizuo** [comment]
tested manually.

**Comment by wuhuizuo** [comment]
fix the issue we found in releasing for v8.4.0

**Comment by wuhuizuo** [comment]
/hold

I want add more changes for other script, i do not want to write more chore pull requests.

**Review by purelind — APPROVED** [review]
/lgtm

File: scripts/artifacts/tag-rc2ga-on-repos.sh [hunk 1]
```diff
@@ -44,6 +44,12 @@ function tag_oci_artifact_repos() {
     "tikv/pd/package"
     "tikv/tikv/package"
   )
+  # binlog is stoped releasing since v8.4.0.
```
Comment (purelind) [inline scripts/artifacts/tag-rc2ga-on-repos.sh:47]: The comparison here may encounter an unexpected situation like v8.10.0 < v8.4.0.

Comment (wuhuizuo) [reply scripts/artifacts/tag-rc2ga-on-repos.sh:47]: I fixed them.

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**